Cargo.toml

```diff
@@ -97,6 +97,7 @@ indicatif = "0.17"
 indoc = "2"
 io_tee = "0.1"
 itertools = "0.14"
+lazy_static = "1.5.0"
```

Comment:
AFAIK you should be able to do the same with `LazyCell` / `LazyLock` / `OnceCell` / `OnceLock` from std instead of this dep in recent Rust (those have been stabilized lately).